README.md

```diff
@@ -28,6 +28,26 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
 
 ### A
 
+#### **Accord Project**
+
+- **Swag**: T-shirt and Stickers
+- **Requirements**: Create pull requests on [Accord Project's](https://accordproject.org) participating repositories.
```

Comment:
There are [very clear requirements](https://www.accordproject.org/events/hacktoberfest-2019/) for each of the swag opportunities, please spell them out. Follow the rules from the Contributing file and the other companies on the list.